projects/components/src/file-display/file-display.module.ts

```diff
@@ -0,0 +1,14 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { FormattingModule } from '@hypertrace/common';
+import { IconModule } from '../icon/icon.module';
+import { ProgressBarModule } from '../progress-bar/progress-bar.module';
+import { TooltipModule } from '../tooltip/tooltip.module';
+import { FileDisplayComponent } from './file-display.component';
+
+@NgModule({
+  imports: [CommonModule, IconModule, ProgressBarModule, FormattingModule, TooltipModule],
```

Comment:
ProgressBarModule doesn't seem to be used. Please remove the ones not needed.

Reply:
Done